### DISCUSSION TITLE
Open Source Contributors - A promising alternative to idea based funding

### DISCUSSION THREAD

**Discussion by lovegrovegeorge** [akash-network/community#512] [General]
![proposal-header](https://github.com/akash-network/community/assets/6073296/0aa15d88-0a0d-48c3-b282-11c533265830)

We’ve recently finished a proposal outlining the opportunity to experiment with an open source contributor funding process. This suggested funding process could become a highly effective and scalable approach for Web3 ecosystems to more consistently generate impact.

The opportunity to experiment with a contributor funding approach is better highlighted by understanding some of the problems that exist with idea based funding:
- For contributors, the proposal submission process can require a large amount of effort and time upfront for contributors to participate and then also handle the ongoing burden of proposal writing and budgeting complexities to get involved in an ecosystem. The structure and incentives of this funding process result in a reduced amount of contribution flexibility and income stability that can deter contributors from participating or limit their ability to easily generate high impact.
- For voters, it is often highly complex to compare and select ideas effectively. Many voters lack the sufficient context, skills and experience required to be well informed and effectively participate in these decisions. The selection process can be highly time consuming and complex for the voters. Voters can rarely express their exact preferences with their voting decisions and also do not have enough accountability or incentives to be expected to spend a meaningful amount of time on voting to make more optimal and well informed decisions.
- For Web3 ecosystems, ideas are often treated as ephemeral yes or no funding decisions rather than being a collaborative process that looks to discuss and explore different solution approaches. In larger funding processes it can also become easier for innovative ideas to be ignored due to being less well understood or known. The allocation of assets can also be more inefficient in situations where the allocated funds are not actively being used to generate contribution outcomes, this can increase the percentage of deadweight assets that are not being fully utilised at a given point in time.

To understand these problems in more detail you can review our analysis on the current funding landscape - https://docs.contributors.org/proposal/open-source-contributors/current-funding-landscape

The good news is that most of these problems can be either greatly reduced or fully resolved! An open source contributor funding process can help with resolving these problems and also could become a highly reliable and effective process for maintaining and improving Web3 ecosystems over the long term.

![web3-open-source-contributor-funding](https://github.com/akash-network/community/assets/6073296/3bb29bbb-bbc0-4b9e-96c0-f9fd376e2c00)

Our proposal outlines the suggestion of experimenting with directly funding a small number of open source developers that would help with developing any open source initiatives - this could include improving any existing pieces of software used in the ecosystem or creating entirely new tools and libraries.

The advantages and long term opportunities for adopting a contributor funding approach are numerous. We’ve covered the advantages and opportunities of this suggested funding process in more detail in our proposal - https://docs.contributors.org/proposal/open-source-contributors/open-source-contributor-funding

You can alternatively watch our videos that cover the same content from our proposal:
- https://youtu.be/ZWo9qNm3mvs - Current Web3 funding landscape
- https://youtu.be/f3fP2bjf6rY - Tech company comparison with Web3
- https://youtu.be/mQHqwHIDKMo - Open source contributor funding

The background analysis that helps to support this proposal can be found here - [https://disbursement.treasuries.co/](https://disbursement.treasuries.co/)

<br/> 

# Experimentation in Akash

The Akash ecosystem has an idea funding processes that can be accessed on their [funding information page](https://akash.network/development/funding-program/). This process could be great for people that already have an idea about what to work on but does not help with situations where developers are keen to join the ecosystem but do not have an idea to execute at the moment.

Experimenting with an open source contributor funding process could simply mean adding in an extra form process to enable software developers to indicate their interest in contributing to the Akash ecosystem as a contributor. These individuals could then help with existing open source solutions being built or new ones that the Akash Foundation suggest to them.

The immediate opportunity with experimenting with a contributor focused funding approach would be that it would make it easier for software developers to indicate their interest in working in the Akash ecosystem. The Akash Foundation could then identify if there are any promising candidates that could be suitable for contributing towards any relevant initiative in the ecosystem. Contributor funding proposals could end up bringing in impactful talent that otherwise might not have got involved due to the more time consuming upfront idea proposal process. The idea process can make it more difficult for these individuals to just express their interest in working in the ecosystem. A contributor focussed process could provide a more collaborative path for identifying promising talent and matching them with ongoing work that could create impact in the ecosystem.

<br />

# Experiment facilitation

If any ecosystem was interested in experimenting with this suggested funding process but doesn’t want to handle the process fully themselves I am eager to collaborate with them and help wherever I can with setting up and running any of these experiments that are focussed on contributor funding.

In either event, contributor funding experiments across many ecosystems could be highly beneficial for the wider industry. The outcomes from these experiments can be analysed to better compare the strengths and weaknesses of this suggested approach against the widely adopted idea based funding approach that we see being more widely used across Web3 ecosystems today. Analysing and documenting these experiment outcomes is something that we intend to work on at the [Web3 Association](https://web3association.co/)!

<br/>

# Community discussion

**General thoughts & feedback**
There’s likely going to be many opportunities and problems that could be better explored and addressed. If you have any immediate thoughts and feedback please share anything below in a comment!

**Organising a wider discussion**
If there are already weekly or monthly organised discussion events that happen internally or publicly I’d be delighted to join one to listen to peoples different perspective about this funding process suggestion to better understand any of the different viewpoints around this approach. Alternatively I could also help to facilitate a one off discussion instead. Please share below if you would be interested in having a dedicated discussion in the near future about this suggested experiment.

**Direct communication**
If anyone would prefer to chat about this funding process suggestion with me directly then please feel free to reach out to me on Discord - lovegrovegeorge or Telegram - https://t.me/georgelovegrove. Otherwise just throw any questions in the comments below!

**Comment by hewi333** [comment]
Hi @lovegrovegeorge, thank you for laying out what you do at Contributors.org. We have a vibrant ecosystem that allows open source contributors to work and earn compensation for their efforts. Everything we do is in the open and you are free to [join](https://akash.network/development/current-groups/) any Special Interest Group (SIG) or Working Group (WG) call as they are all public. You can also view work and recorded meetings [here](https://github.com/akash-network/community). Finally, you can check out our [best practices](https://github.com/orgs/akash-network/discussions/170) for submitting governance proposals.

Akash is built using Cosmos SDK, which gives us quite a robust governance mechanism and our [community pool](https://www.mintscan.io/akash/address/community-pool) is currently healthy. You can see past proposals [here](https://www.mintscan.io/akash/proposals).

I do like the idea of creating an intake form for those that want to "opt-in" and where they want to contribute. At this time, do you have any specific funding requests or are you just getting the lay of the land?

**Reply by lovegrovegeorge** [reply]
Hey @heyhewi, thanks for the response. This proposals purpose is to outline the reasons why a contributor focussed funding process is so promising for the short and long term.

I'm not requesting any funding at the moment, just trying to identify the ecosystems that would be interest in this suggested funding process and do anything I can to help them.

How are the special interest group and working groups funded or is it volunteer? If funded, does the Akash team select who is funded to be in those groups? Keen to learn more there, couldn't find any answers immediately on page.
